TEMPLATE-config.toml

```diff
@@ -1,15 +1,54 @@
-[bot]
-prefix = ''
+# Discord bot token
 token = ""
-ownerID = 
-status = ''
-error_channel_id = XXXXXXXXXXXXXXXXXX
```

Comment:
> Discord IDs should be stored as strings, not int or long.

As per the [pycord docs](https://docs.pycord.dev/en/master/api.html?highlight=get_channel#discord.Bot.get_channel), channel IDs should be stored as ints for using the Bot.get_channel() method.

> Also, you've got _id appended to each of these keys so ideally these should all be key = "" instead of having tons of placeholder X's.

You are again looking at removed code. Please see how they are [currently named](https://github.com/MiscGuild/MiscBot/pull/48/files#diff-d0cccd9da75985febe381f688c26fc2676809145714edb5c9210cbe7bb9866c6R16) (X's will be fixed.)